oshc/main/migrations/0001_initial.py

```diff
@@ -16,14 +16,37 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='chatSession',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
-                ('title', models.CharField(help_text='Session title', max_length=128)),
-                ('profile_name', models.CharField(help_text="The person's name", max_length=128)),
-                ('profile_url', models.URLField(help_text="The Url of the person's website")),
-                ('description', models.TextField(help_text='Session details', max_length=512)),
+                ('id', models.AutoField(auto_created=True,
+                                        primary_key=True,
+                                        serialize=False, verbose_name='ID')),
+                ('title', models.CharField(help_text='Session title',
```

Comment:
Nope. I had to manually do it like that so that hound ci error doesn't come. With hound ci, you'll get something like:
`('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),`

Reply:
Thanks for reviewing @tapasweni-pathak,
So what I'm trying to do now is adding up a boolean field in the contest model which will be having a  default value 0. So, when the user  will submit the contest, the admin can then review it and if he/she finds that this contest can be put up on the website then he/she can change the boolean field to 1 and the contest will be displayed on the website. Is it okay?

Reply:
Yeah, cool. Please go ahead.